baseplate/lib/timer.py

```diff
@@ -0,0 +1,56 @@
+from enum import Enum
+from enum import unique
+from typing import Callable
+from typing import Optional
+
+from schedule import CancelJob
+
+from baseplate.lib.threaded_scheduler import ThreadedScheduler
+
+
+@unique
+class TimerState(Enum):
+    Running = "Running"
+    Stopped = "Stopped"
+
+
+class Timer:
+    def __init__(
+        self,
+        function: Callable[[], None],
+        interval: float,
```

Comment:
I think, in general using `timedelta`s rather than raw floats/ints makes for a cleaner interface?

Reply:
I've updated everything to use `timedelta`s